JenkinsFile

```diff
@@ -55,6 +55,14 @@ pipeline {
           }
         }
 
+        stage('typescript') {
+          steps {
+            sh("yarn tsc --noEmit")
+            script {
+              FAILED_STAGE = 'lint'
```

Comment:
Should we add this to the existing lint stage, or update the FAILED_STAGE value to the correct one?

Reply:
good catch I will change it to `typescript`